src/robot/errors.py

```diff
@@ -230,7 +233,6 @@ def _get_attrs(self, errors):
     def get_errors(self):
         return self._errors
 
-
```

Comment:
We use two empty lines between classes as PEP-8 recommends. Please add this line and the line a bit more down back.

Reply:
OK, done.